src/swagger/trip.swagger.js

```diff
@@ -357,3 +357,71 @@
  *         description: Trip ID doesn't exists
  *
  */
+
+/**
+ * @swagger
+ * definitions:
+ *   Edit Pending Trip Request:
+ *     type: object
+ *     properties:
+ *       originId:
+ *         type: integer
+ *       destinationId:
+ *         type: integer
+ *       departureDate:
+ *         type: string
+ *       returnDate:
+ *         type: string
+ *       travelReasons:
+ *         type: string
+ *       accommodationId:
+ *         type: integer
+ */
+
+/**
+ * @swagger
+ * /api/trips/requests/{tripId}/edit-trip-request:
+ *   patch:
+ *     tags:
+ *       - trips
+ *     name: Edit Trip Request
+ *     summary: User should be able edit pending trip request
+ *     produces:
+ *       - application/json
+ *     consumes:
+ *       - application/json
+ *     parameters:
+ *       - in: header
+ *         name: Authorization
+ *         required: true
+ *         type: string
+ *       - name: tripId
+ *         in: path
```

Comment:
Parameter in path are always `required:true`.  [https://swagger.io/docs/specification/describing-parameters/](https://swagger.io/docs/specification/describing-parameters/) the section named Required and Optional Parameters

Reply:
Yes. did it